### PR TITLE
changelog: Fix duplicate entries

### PR DESCRIPTION
**Pull request by phlax**
and add a basic yamllint file

partial fix for #23348 and #22222 

Signed-off-by: Ryan Northey <ryan@synca.io>

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

**Comment by phlax** [comment]
cc @wrowe

**Review by wrowe — APPROVED** [review]
lgtm

**Review by wrowe — CHANGES_REQUESTED** [review]
I'm a bit confused, why are you adding yamllint and adding it to gitignore? Seems like the commit is a little too iterative and you hadn't mopped up in between?

**Comment by phlax** [comment]
> I'm a bit confused, why are you adding yamllint and adding it to gitignore?

more like removing it from .gitignore - the default is to ignore `.` prefixed files/dirs

File: .gitignore [hunk 1]
```diff
@@ -16,6 +16,7 @@
 !\.gitignore
 !\.python-version
 !\.style.yapf
+!\.yamllint
```
Comment (phlax) [inline .gitignore:19]: @wrowe - note that the ignore is prefixed with `!` - ie unignore

Comment (wrowe) [reply .gitignore:19]: Doh, thanks.

**Review by wrowe — APPROVED** [review]

**Comment by phlax** [comment]
also worth noting is that this doesnt add yamllint - just a lint conf file that works with the changelog dir

**Pull request merged**